manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -73,6 +73,31 @@ metadata:
   name: vsphere-csi-node
   namespace: vmware-system-csi
 ---
+kind: ClusterRole
```

Comment:
Why do we have two roles now for CSI node?

Reply:
We had a `Role` for node till now. I am introducing a `ClusterRole` in this PR